### ISSUE TITLE
Feature showcases: 'Sorting' information is not announced by screen readers for ‘Field’ column header.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#86657]
### What happened?

'Sorting' information is not announced by screen readers for ‘Field’ column header button. Screen reader does not announce anything for the control, after activating the button. 
<img width="952" alt="code snippet-'Sorting' information is not announced by screen readers for ‘Field’ column header" src="https://github.com/grafana/grafana/assets/93735775/0757051b-5273-4cbc-8b74-a001a4021413">

https://github.com/grafana/grafana/assets/93735775/086c1158-f4f9-4ef0-8935-1fe75ac11364

fc96-14be-48ba-bae6-be19462358df">

### What did you expect to happen?

Screen reader should announce as 'Field, activate to sort column ascending/descending order' when user activates it.

### Did this work before?

NO

### How do we reproduce it?

1. Open the URL [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1) in edge Dev browser.
2. ‘Grafana Play Home’ page will open. 
3. Navigate to ‘Home’ link & activate it. 
4. Select ‘Gradient color modes’ from the ‘Feature showcases’. 
5. Turn on narrator using ‘Ctrl+Windows+Enter’ keys. 
6. Navigate to ‘field’ column header & press enter. 
7. Observe the screen reader narration.

### Environment (with versions)?

Grafana: v11.1.0-69372 (51da96d94e)
OS: 24H2 (OS Build 26100.1) 
Browser: Edge version 125.0.2506.0 (Official build) dev (64-bit)

### Grafana platform?

None

**Comment by msftedad** [comment]
Closing this issue as we have closed our bug.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by gtk-grafana** [comment]
Duplicate of https://github.com/grafana/grafana/issues/117842, should be fixed with the latest version of the Table

**Comment by fastfrwrd** [comment]
Fix for this issue should be available as far back as Grafana 12.2.0.
